addons/collectd/0.0.1/Manifest

```diff
@@ -0,0 +1,4 @@
+
+apt collectd
+yum epel-release
```

Comment:
This would ship `epel-release` to customers. It should instead be added to https://github.com/replicatedhq/kURL/blob/master/bin/save-manifest-assets.sh#L83 and https://github.com/replicatedhq/kURL/blob/master/bin/save-manifest-assets.sh#L94.